crates/icrate/CHANGELOG.md

```diff
@@ -56,60 +63,65 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   - `UniformTypeIdentifiers`
   - `UserNotifications`
   - `WebKit`
-* Updated the SDK version from XCode `14.0.1` to `14.2`.
+- Updated the SDK version from XCode `14.0.1` to `14.2`.
```

Comment:
I appreciate the desire to make the changelogs more "standard", but please do so in a separate PR (and do it for the other changelogs as well).

Reply:
Oh, that was an unintentional format-on-save that I didn't notice. Sorry.